apps/src/templates/certificates/Congrats.jsx

```diff
@@ -67,6 +116,15 @@ export default function Congrats(props) {
     curriculumUrl,
   } = props;
 
+  // Determine what time we should consider the current time to be when
+  // rendering the page. This is useful for testing UI changes which happen
+  // at particular times of the year.
+  // the currentDate can be overridden using the query string param 'currentDate'
+  const currentDateOverride =
+    hasQueryParam('currentDate') && new Date(queryParams('currentDate'));
+  // default to the real current time.
+  const currentDate = currentDateOverride || props['currentDate'] || new Date();
+
```

Comment:
I like having the ability to override the current date.

Reply:
Yeah this should probably be a pattern we should start following. Maybe this should actually be in redux so all components can easily adopt it. Something to think about. I won't do it in this PR.